app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestViewModel.kt

```diff
@@ -82,10 +75,6 @@ class MeetingRequestViewModel(
   fun onMeetingRequestChange(newMessage: String) {
     meetingRequestState = meetingRequestState.copy(message = newMessage)
   }
-  /** Sets the status of the message, shows that it is ready to be sent */
-  fun onSubmitMeetingRequest() {
-    meetingRequestState = meetingRequestState.copy(isEnteringMessage = false)
-  }
 
   /** Sends the message from our user to the target user */
   fun sendMessage() {
```

Comment:
Important: Both sendMessage and sendMeetingRequest perform nearly identical tasks, differing only by the Firebase Function name. This duplication could be refactored into a single reusable method.